### ISSUE TITLE
Batch Normalization 넣기

### ISSUE THREAD

**Issue by ssaru** [DeepBaksuVision/YOLOv1#3]

**Comment by ssaru** [comment]
2018-09-01
모든 Layer에 Batch Normaliation 값을 넣었을 때, Loss값 nan 발생
Batch norm을 2개 (첫번째와 모델 중간)에 넣었을 때, 혹은 Batch norm을 1개(첫번째)에 넣었을 때만
수렴하는 형태를 보임.

분석이 필요해보임

- 분석사항
1. Dataloader가 제대로 데이터셋을 불러오는지 확인
2. Batch norm에 대해서 복기 한번 더 해볼 것
